alerter/TemperatureThresholdManipulator.java

```diff
@@ -0,0 +1,3 @@
+public interface TemperatureThresholdManipulator {
+     Integer networkAlertStub(float celsius);
```

Comment:
Name of the interface method indicates that it is a stub, but in reality, it is a just an API, which is either overridden by test code or production code. Please correct the naming.